Update limitations to include checkboxes for common project configurations (#8)
vanderbilt-redcap/external_module_template: README.md

<!-- Mention any limitations to this module that users may expect, e.g. won't work with repating instances or events, won't work with signature fields -->

> If **checked**, you can expect the module to account for these common project variants:

Author: This feels a little clunky to me so I'd happily entertain changes to this sentence.

> If **checked**, you can expect the module to account for these common project variants:

Reviewer: ```suggestion
If **checked**, this module has been developed to account for any variation in the following:
```

- [ ] Longitudinal projects
- [ ] Repeating events
- [ ] Repeating instances

> 

Reviewer: ```suggestion
[ ] Field names/form names (if unchecked, this module will not work without specific field names/form names on the enabled project)
```
I hear what you're saying about hard-coded fields, but sometimes it might still happen and we should include that checkbox.
